<launch>
  <node pkg="advanced_navigation_driver" name="advanced_navigation_driver" type="advanced_navigation_driver" output="screen" args="_port:=/dev/ttyUSB0 _baud:=115200">
  </node>
</launch>
